pkg/controller/uniteddeployment/subset.go

```diff
@@ -47,12 +47,25 @@ type SubsetStatus struct {
 	ReadyReplicas        int32
 	UpdatedReplicas      int32
 	UpdatedReadyReplicas int32
+	UpdatedRevision      string
 	UnschedulableStatus  SubsetUnschedulableStatus
 }
 
+// SubsetUnschedulableStatus stores the unschedulable status of the Subset, which is used by adaptive strategy.
 type SubsetUnschedulableStatus struct {
-	Unschedulable bool
-	PendingPods   int32
+	Unschedulable           bool // has reserved pods
+	PreviouslyUnschedulable bool // has condition Schedulable == False
```

Comment:
plz comment why Unschedulable is not enough and how PreviouslyUnschedulable is used